projects.html

```diff
@@ -0,0 +1,239 @@
+<link href="https://fonts.googleapis.com/css2?family=Arima:wght@100..700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<style>
+  
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+  margin: 0;
+  padding: 0;
+  background-image: url(images/TechGirlsHub\ bg.jpg);
+  background-size: cover;
+  background-position: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  
+  }
+  
+  .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0;
+  background: linear-gradient(90deg, #00a3c8 0%, #d282fc 100%);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  }
+  
+  .header .logo {
+  width: 200px;  
+  height: auto;
+  max-height: 80px;
+  margin-right: 10px;
+  background-color: transparent;
+}
+ 
+
+
+  
+  .container {
+    margin-top: 50px;
+    padding: 20px;
+    background-color: #B3C4D8;
+    border-radius: 8px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  }
+  
+    .projects h2 {
+    font-size: 32px;
+    color:black;
+    margin-bottom: 10px;
+  }
+  
+  .projects p {
+    font-size: 18px;
+    color: black;
+    margin-bottom: 40px;
+  }
+  
+ 
+  .card {
+    background-color:#CB83FB;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    margin: auto;
+  }
+  
+  .card img {
+    border-radius: 8px 8px 0 0;
+    width: 100%;
+  }
+  
+  .card-body {
+    padding: 20px;
+    text-align: center;
+  }
+  
+  .card-title {
+    font-size: 24px;
+    color: black;
+    margin-bottom: 10px;
+  }
+  
+  .card-text {
+    font-size: 16px;
+    color: black;
+    margin-bottom: 20px;
+  }
+  
+  .btn {
+    background-color:#B80E65;
+    color: black;
+  }
+  
+  
+  .carousel-item .btn {
+    margin-bottom: 30px;
+  }
+  
+ 
+  .carousel-indicators {
+    margin-bottom: 80; 
+  }
+  
+  .carousel-indicators button {
+    background-color: #B80E65;
+    width: 10px;
+    height: 10px;
+    border-radius: 50%;
+    margin: 0 3px;
+  }
+  
+  .carousel-indicators .active {
+    background-color: #B3C4D8;
+  }
+  
+ 
+  .footer {
+    background: linear-gradient(90deg, #00a3c8 0%, #d282fc 100%);
+    padding: 20px 0;
+    color:#B80E65;
+    text-align: center;
+  }
+  
+  .footer p {
+    margin: 0;
+    font-size: 14px;
+    font-weight: 700;
+  }
+  
+</style>
+<body>
+
+  
+  <header class="header">
+    <div class="container d-flex justify-content-between align-items-center">
+      <img src="images/IMG-20241022-WA0006.png" alt="TechGirlsHub Logo" class="logo">
+      <nav>
+        <ul>
+          <li><a href="#about">About Us</a></li>
+          <li><a href="#services">Services</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#team">Team Members</a></li>
+          <li><a href="#contact">Contact Us</a></li>
```

Comment:
please add the link for Feedback Rating, the href for the file is "feedbackRating.php". 

Example:
<li><a href="feedbackRating.php">Feedback Rating</a></li>